modules/process/manager.go

```diff
@@ -101,7 +100,10 @@ func (pm *Manager) ExecDirEnv(timeout time.Duration, dir, desc string, env []str
 	stdOut := new(bytes.Buffer)
 	stdErr := new(bytes.Buffer)
 
-	cmd := exec.Command(cmdName, args...)
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
```

Comment:
What happens if the `timeout` is negative? (Since the functions docs says to use `-1 for DefaultTimeout` (which we don't seems to honour... )

Reply:
@bkcsoft The first line on this function has checked that. if timeout == -1 timout = 60 *time.Second

Reply:
Do you want 
```
if timeout < 0 {
    timeout = 60 * time.Second
}
``` 
?